nodewatcher/nodewatcher.py

```diff
@@ -146,6 +144,23 @@ def maintainSize(asg_name, asg_conn):
         log.debug('capacity less then or equal to min size.')
         return True
 
+
+def saveIdleTime(persisted_data):
+    try:
+        if not os.path.exists(_NW_DATA_DIR):
+            os.makedirs(_NW_DATA_DIR)
+    except OSError as ex:
+        if ex.errno == errno.EEXIST and os.path.exists(_NW_DATA_DIR):
```

Comment:
I think we can safely get rid of this check and just log and raise the exception in case of errors

Reply:
Okay